node/src/network.rs

```diff
@@ -0,0 +1,97 @@
+use std::str::FromStr;
+
+use celestia_types::hash::Hash;
+#[cfg(not(target_arch = "wasm32"))]
+use clap::ValueEnum;
+use hex::FromHexError;
+use libp2p::core::multiaddr::Error as MultiaddrError;
+use libp2p::Multiaddr;
+use serde_repr::{Deserialize_repr, Serialize_repr};
+use thiserror::Error;
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
+
+#[cfg_attr(not(target_arch = "wasm32"), derive(ValueEnum))]
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Serialize_repr, Deserialize_repr)]
+#[repr(u8)]
+pub enum Network {
```

Comment:
btw celestia introduced mainnet already, but we can be slow to the party I guess as it hasn't started yet
https://github.com/celestiaorg/celestia-node/blob/main/nodebuilder/p2p/bootstrap.go#L40